code/frameworks/nextjs/src/config/webpack.ts

```diff
@@ -54,3 +56,23 @@ const setupRuntimeConfig = (baseConfig: WebpackConfig, nextConfig: NextConfig):
 
   baseConfig.plugins?.push(new DefinePlugin(definePluginConfig));
 };
+
+export const applyNextConfigWebpackConfig = (
+  baseConfig: WebpackConfig,
+  nextConfig: NextConfig
+): void => {
+  if (typeof nextConfig.webpack === 'function') {
+    nextConfig.webpack(baseConfig, {
+      dev: process.env.NODE_ENV === 'development',
+      isServer: false,
+      buildId: nextConfig.buildId || '',
+      dir: '.',
+      config: nextConfig as NextConfigComplete,
+      defaultLoaders: {
+        babel: {},
+      },
```

Comment:
This is tricky since the loader we are using is not the same as the one what Next.js uses. But you can do something like this:

```
defaultLoaders: {
  babel: createBabelLoader(...)
```

createBabelLoader is coming from https://github.com/storybookjs/storybook/blob/next/code/builders/builder-webpack5/src/preview/loaders.ts#L8C3-L8C3